packages/valory/skills/decision_maker_abci/behaviours/sell_outcome_token.py

```diff
@@ -58,7 +57,20 @@ def market_maker_contract_address(self) -> str:
     @property
     def outcome_index(self) -> int:
         """Get the index of the outcome for which the service is going to sell token."""
-        return cast(int, self.synchronized_data.vote)
+        return cast(int, self.synchronized_data.previous_vote)
+
+    @property
+    def return_amount(self) -> int:
+        """Get the amount expected to be returned after the sell tx. """
+        previous_vote = self.synchronized_data.previous_vote
+
+        if previous_vote == 0:
+            return self.sampled_bet.invested_amount_yes
+
+        else:
+            return self.sampled_bet.invested_amount_no
```

Comment:
You cannot know if index 0 will always be "yes".

You could extract the following logic to a separate method and reuse it here:
https://github.com/valory-xyz/trader/blob/a3aae14f6a2960333c27c6df1a8f1cbdedce795a/packages/valory/skills/market_manager_abci/bets.py#L290-L294